sklearn/exceptions.py

```diff
@@ -135,7 +135,14 @@ class FitFailedWarning(RuntimeWarning):
     ...         gs.fit(X, y)  # This will raise a ValueError since C is < 0
     ...     except ValueError:
     ...         pass
-    ...     print(repr(w[-1].message))
+    ...     msg = repr(w[-1].message)
+    ...     # if msg.find(r'Traceback (most recent call last):\\n') > -1:
+    ...     ct = msg.count('\\n')
+    ...     if ct > 1 or msg.count(r'\\n') > 1:  # has stacktrace
```

Comment:
Instead of cleaning the output, just use ellipsis in the doctest

Reply:
Sounds good.